### ISSUE TITLE
Accessibility: Combobox: Group headers within the combo box list are not read.  

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3454]
Description: Group headers within the combo box list are not read

Expected: Group headers within the combo box list should be read out to the user

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94522601-84e0e280-024d-11eb-844a-a088d754fb67.png)

![image](https://user-images.githubusercontent.com/32538291/94522616-88746980-024d-11eb-9669-1b57a675c201.png)

**Comment by puru-hk** [comment]
Focus is provided for the header, but no speech output

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue still persists.
The fruits and Vegetables is not read by the screen reader.
So please fix the issue.

Thanks and Regards
Sanjay

**Comment by Sanjay-1312** [comment]
Hello Colleagues,

The issue still persists.
The groups name  like fruit, Vegetables , Types of Fruits and Types of Vegetables is not read.
The focus is available on the group elements but its not read.
ALT+ down arrow to open the comboboxes.
So please fix the issue.
URL: https://fundamental-ngx.netlify.app/#/platform/combobox
JAWS 2021

![image](https://user-images.githubusercontent.com/78714518/119932271-91663800-bfa0-11eb-82f3-9f16f85d9111.png)
![image](https://user-images.githubusercontent.com/78714518/119932342-b8bd0500-bfa0-11eb-8067-89180fff5bc2.png)

**Comment by Betrozov** [comment]
Hi @Sanjay-1312 
please test using this link
https://deploy-preview-5206--fundamental-ngx.netlify.app/

**Comment by Sanjay-1312** [comment]
Hello colleagues,

The Issue still persists i can able to reproduce the issue.
The list header such as 'Vegetable' and 'Fruits' is not read
 when accessed it is read as 'Open List box' when 'Vegetable is accessed and nothing is read on accessing the 'Vegetable' Header.

![image](https://user-images.githubusercontent.com/78714518/120317816-bb9b5b00-c2fc-11eb-8a74-3d37aa3eb7b0.png)


URL:
https://deploy-preview-5206--fundamental-ngx.netlify.app/

JAWS 2021

**Comment by puru-hk** [comment]
@Betrozov Please see the above comments from the accessibility team.

**Comment by OleksiyDrozdov** [comment]
Duplicating #[3456](https://app.zenhub.com/workspaces/fundamental-library-60761c0894e25c0011e828b4/issues/sap/fundamental-ngx/3456)
